### ISSUE TITLE
[2.13.0-M4] missing method in IterableFactory

### ISSUE THREAD

**Issue by MasseGuillaume** [scala/bug#10975]
```
def concat[A](xss: Traversable[A]*): CC[A]
```

from scala.collection.generic.GenTraversableFactory

**Comment by julienrf** [comment]
Note that this method was not available in the `Set` factory object in 2.12. If we add it to `IterableFactory` then it will become available to the new `Set` factory objects. I think this is fine, though.

The new signature should be:

~~~ scala
def concat[A](xss: IterableOnce[A]*): CC[A]
~~~

**Comment by gmethvin** [comment]
It also did not exist before on `Iterator`, but I think it's also fine there.
